lms/static/sass/views/_verification.scss

```diff
@@ -1443,7 +1448,7 @@
 
       border-top: ($baseline/4) solid $m-gray-d4;
       padding: $baseline ($baseline*1.5);
-      background: $m-gray-l4;
+      background: $light-300;
```

Comment:
I also don't think we can use brand here unless we import them? the question is if we should import them here?

@abutterworth explained to me that we shouldn't.

Reply:
Just to keep other in the loop - see below comment!